js/signin.js

```diff
@@ -68,20 +78,28 @@ pwInput.addEventListener("focusout", function () {
 // form submit (enter, submit click)
 signinForm.addEventListener("submit", function (e) {
   e.preventDefault();
-  if (emailInput.value === "test@codeit.com" && pwInput.value === "codeit101") {
-    window.location.href = "/folder";
-  } else {
-    inputFocusBorderChange(
-      "valid_error",
-      emailInput,
-      emailValidText,
-      "이메일을 확인해 주세요."
-    );
-    inputFocusBorderChange(
-      "valid_error",
-      pwInput,
-      pwValidText,
-      "비밀번호를 확인해 주세요."
-    );
-  }
+  const id = emailInput.value;
+  const pw = pwInput.value;
+
+  postReqeustApi("api/sign-in", {
+    email: id,
+    password: pw,
+  })
```

Comment:
아마 fetch함수가 많이 반복되어서 postRequestAPI로 만들어 두신것 같아요 ! 
아주 잘 하셨어요. 조금만 더 개선 하려면 Post라고 못박아 두는것보다
HTTP METHOD를 파라미터로 받을 수 있도록 하면 좋을 것 같아요.

```suggestion
  ReqeustApi("api/sign-in","POST", {
    email: id,
    password: pw,
  })
```